evap/staff/importers.py

```diff
@@ -324,6 +336,9 @@ def write_enrollments_to_db(self, semester, vote_start_datetime, vote_end_date):
                 course = Course.objects.get(semester=semester, name_de=course_data.name_de)
                 # This is safe because the user's email address is checked before in the importer (see #953)
                 student = UserProfile.objects.get(email=student_data.email)
+                if not student.is_active:
+                    student.is_active = True
+                    student.save()
```

Comment:
isn't this sort of contained in one of the `self.make_users_active(user_list)` below?